src/sentry/utils/performance_issues/performance_detection.py

```diff
@@ -536,9 +538,15 @@ def _is_blocking_render(self, span):
         fcp_ratio_threshold = self.settings.get("fcp_ratio_threshold")
         return span_duration / self.fcp > fcp_ratio_threshold
 
-    def _fingerprint(self, span):
-        hashed_spans = fingerprint_spans([span])
-        return f"1-{GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN.value}-{hashed_spans}"
+    def _fingerprint(self, span: Span):
+        url_hash = self._url_hash(span)
+        return f"1-{GroupType.PERFORMANCE_RENDER_BLOCKING_ASSET_SPAN.value}-{url_hash}"
+
+    def _url_hash(self, span: Span) -> str:
+        url = urlparse(span.get("description") or "")
+        stripped_path = ASSET_HASH_REGEX.sub("%s", url.path)
```

Comment:
I'm not sure if this will come into play, but FYI the current transaction name parameterization uses `*` not `%s`, so that might be safer to use here, in case we plug in parameterization for asset URLs